api/server/api-routes.js

```diff
@@ -10,160 +10,124 @@ router.get("/", function (req, res) {
 });
 // Import user controller
 let userController = require("./controllers/userController");
-let groupController = require("./controllers/groupController")
+let groupController = require("./controllers/groupController");
+
 // Import meeting controller
 var meetingController = require("./controllers/meetingController");
 
 // User routes
 
 // Create user
-router
-  .route("/users")
-  .post(userController.register);
+router.route("/user/create").post(userController.register);
+
+//Login user
+router.route("/user/login").post(userController.login);
 
 // Get user by Id
 router
-  .route('/user/:userId')
+  .route("/user/:userId")
   .get(userController.view)
+  .delete(userController.delete)
+  .patch(userController.update);
 
-// Get user groups 
-router
-  .route('/user/groups/:userId')
-  .get(userController.userGroups);
+//Get user groups
+router.route("/user/groups/:userId").get(userController.userGroups);
 
-// Get users by username 
-router
-  .route('/users/:username')
-  .get(userController.usersByUsername)
+//Get users by username
+router.route("/users/:username").get(userController.usersByUsername);
```

Comment:
I agree with these changes but they will break the frontend GroupController that sends the data in the body. Do not merge to master unless you change the GroupController in the frontend.

Reply:
> I agree with these changes but they will break the frontend GroupController that sends the data in the body. Do not merge to master unless you change the GroupController in the frontend.

Apoorva has approved the changes, we can't merge because we'll break a front end controller